pyogrio/tests/test_geopandas_io.py

```diff
@@ -187,40 +190,105 @@ def test_read_fids_force_2d(test_fgdb_vsi):
 
 
 @pytest.mark.parametrize(
-    "driver,ext",
-    [
-        ("ESRI Shapefile", "shp"),
-        ("GeoJSON", "geojson"),
-        ("GeoJSONSeq", "geojsons"),
-        ("GPKG", "gpkg"),
-    ],
+        "suffix",
+        [".fgb", ".geojson", ".geojsons", ".gpkg", ".json", ".shp", ],
 )
-def test_write_dataframe(tmpdir, naturalearth_lowres, driver, ext):
-    expected = read_dataframe(naturalearth_lowres)
-
-    filename = os.path.join(str(tmpdir), f"test.{ext}")
-    write_dataframe(expected, filename, driver=driver)
-
-    assert os.path.exists(filename)
-
-    df = read_dataframe(filename)
-
-    if driver != "GeoJSONSeq":
-        # GeoJSONSeq driver I/O reorders features and / or vertices, and does
-        # not support roundtrip comparison
+def test_write_dataframe(tmp_path, naturalearth_lowres, suffix):
+    input_gdf = read_dataframe(naturalearth_lowres)
+    assert isinstance(input_gdf, gp.GeoDataFrame)
+    output_path = tmp_path / f"test{suffix}"
+
+    if suffix == ".fgb":
```

Comment:
This suggests FGB should be split into a separate test case in order to probe at this specific behavior.